progress_bar_circular.go

```diff
@@ -0,0 +1,204 @@
+package unichart
+
+import (
+	"io"
+	"math"
+
+	"github.com/unidoc/unichart/mathutil"
+	"github.com/unidoc/unichart/render"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+// CircularProgressBar is a component that will render circular progress bar component.
+type CircularProgressBar struct {
+	BackgroundStyle render.Style
+	ForegroundStyle render.Style
+	LabelStyle      render.Style
+	ColorPalette    render.ColorPalette
+
+	// If the value is true then the progress bar would rendered counter clockwise.
```

Comment:
Probably should be
// Reversed if the value is `true` then the progress bar would rendered counter clockwise.